supervisord/clamav_reusable.conf

```diff
@@ -0,0 +1,5 @@
+[program:clamav-update]
+command=bash -c "if [ \"${CLAM_AV}\" = 'yes' ]; then /usr/bin/freshclam -d \
```

Comment:
As long as you're doing this patch, perhaps you could move the handling of `CLAM_AV` to the backend entrypoint script? If `CLAM_AV` is not `yes`, then the script could avoid including this file at all.